resources/ext/ezbake.conf

```diff
@@ -21,7 +21,8 @@ ezbake: {
                          "mkdir -p /etc/puppet/manifests"],
                install: ["echo \\\"os-settings: {\\\"                         > $DESTDIR/$projconfdir/conf.d/os-settings.conf",
                          "echo \\\"    ruby-load-path: [$rubylibdir]\\\"      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf",
-                         "echo \\\"}\\\"                                      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf" ]
+                         "echo \\\"}\\\"                                      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf" ],
+               pre-start-action: ["install --group={{user}} --owner={{user}} -d /var/run/puppet"]
```

Comment:
With this pre-start-action, I get a systemd config for the puppetserver.service which has:

~~~~
ExecStartPre=install --group=puppet --owner=puppet -d /var/run/puppet
~~~~

I tested out an ezbake package on CentOS 7.  If I delete the `/var/run/puppet` directory before starting the service, the directory doesn't end up getting created and so puppetserver blows up.  If I modify the puppetserver.service to use this instead ...

~~~~
ExecStartPre=/usr/bin/install --group=puppet --owner=puppet -d /var/run/puppet
~~~~

... then the directory is created fine the next time the service starts and the service comes to ready properly.

This leads me to believe that we need to use an absolute path here -- `/usr/bin/install` instead of just "install".  "install" lives under `/usr/bin` by default in the distros I tried -- CentOS 6, CentOS 7, and Debian 7.

@haus - does this path make sense to you?